src/main/java/org/fungover/haze/HazeList.java

```diff
@@ -5,6 +5,7 @@
 
 public class HazeList {
 
+    final List<String> elements;
```

Comment:
I think this field elements shouldn't be here or used? Whit this implementation only one list can be saved in the server? Instead use hazeDatabase to store a list under a key value. Take a look at the existing functions like lpush that uses two helper methods, getValueAsList and listValueAsString to store the list as a key, value in database objekt.